python/python/ecl/ecl/ecl_grid.py

```diff
@@ -386,6 +386,21 @@ def get_name(self):
         n = self._get_name()
         return str(n) if n else ''
 
+    def cell(self, global_index=None, active_index=None, i=None, j=None, k=None):
+        if global_index is not None:
+            return Cell(self, global_index)
+        if active_index is not None:
+            return Cell(self, self.global_index(active_index=active_index))
+        if i is not None:
+            return Cell(self, self.global_index(ijk=(i,j,k)))
+
+    def __getitem__(self, global_index):
```

Comment:
This could check if the index argument was a tuple, i.e. ijk and then calculate global index from that - that way we could also support: `cell = grid[1,2,3]` - which looks kind of nice.

Reply:
Done